src/server/models/users.model.js

```diff
@@ -81,6 +81,7 @@ let userSchema = new Schema({
   job_position: String,
   about: String,
   languages: String,
+  requestGet: Boolean,
```

Comment:
This is for decision whether user get the mentor request or not.
If value is false, user will not receive any mentor request.